src/statement_generator.cpp

```diff
@@ -532,6 +534,7 @@ unique_ptr<TableRef> StatementGenerator::GenerateSubqueryRef() {
 	unique_ptr<SelectStatement> subquery;
 	{
 		StatementGenerator child_generator(*this);
```

Comment:
you can assign `verification_enabled` here in the constructor

Reply:
I created one more constructor to assign `verification_enabled` using it (and to don't search and add the second parameter everywhere in the code).